fv3config/_datastore.py

```diff
@@ -166,5 +168,47 @@ def extract_data():
         shutil.move(os.path.join(app_data_dir, filename_root), local_archive_dir)
 
 
+def is_gsbucket_url(path):
+    if path.startswith(gs_bucket_prefix):
```

Comment:
The if statement here isn't necessary. `return path.startswith(gs_bucket_prefix)` is sufficient.

Reply:
Good call. Changed.